source/opt/function.h

```diff
@@ -65,13 +69,26 @@ class Function {
   void ToBinary(std::vector<uint32_t>* binary, bool skip_nop) const;
 
  private:
-  Module* module_;        // The enclosing module.
-  Instruction def_inst_;  // The instruction definining this function.
-  std::vector<Instruction> params_;  // All parameters to this function.
-  std::vector<BasicBlock> blocks_;   // All basic blocks inside this function.
-  Instruction end_inst_;             // The OpFunctionEnd instruction.
+  // The enclosing module.
+  Module* module_;
+  // The instruction definining this function.
```

Comment:
Minor: "The OpFunction instruction that begins the definition of this function."
(not blocking)